content/en/docs/architecture/jira.md

```diff
@@ -22,6 +22,22 @@ To rerun the validations, comment `/jira refresh` on the PR. This is mostly usef
 All Openshift bugs must be made in the `OCPBUGS` project. For PRs referencing issues from other projects, a link will be
 created but the rest of the lifecycle management will not be performed for the issue.
 
+## Premerge Verification
+
+The Jira GitHub integration provides a method to mark a PR as `verified` before the PR is merged. This allows the integration
```

Comment:
Detail about if there are multiple PRs linked to the Jira issue?

Reply:
If there are multiple PRs for an issue, only the last merged PR has to be marked as verified. I will update the documentation to include that detail.